packages/hw-app-ada/src/Ada.js

```diff
@@ -19,13 +19,13 @@
 import type Transport from "@ledgerhq/hw-transport";
 import { TransportStatusError } from "@ledgerhq/hw-transport";
 
-const CLA = 0xD7;
+const CLA = 0xd7;
 const HARDENED = 0x80000000;
 
 const INS_GET_VERSION = 0x00;
 const INS_GET_EXT_PUBLIC_KEY = 0x10;
 const INS_ATTEST_UTXO = 0x20;
-const INS_RUN_TESTS = 0xF0;
+const INS_RUN_TESTS = 0xf0;
```

Comment:
Hmm, I am wondering whether we shouldn't replace this with
const INS = {GET_VERSION = 0x00, ...} enum object. No needed in this PR though

Reply:
yes, might do in next PR